cmd/notary/delegations.go

```diff
@@ -38,8 +38,8 @@ var cmdDelegationAddTemplate = usageTemplate{
 	Long:  "Add a keys to delegation using the provided public key PEM encoded X509 certificates in a specific Global Unique Name.",
 }
 
-var paths []string
-var removeAll, removeYes bool
+var addPaths, removePaths []string
```

Comment:
I'll make them one var -- earlier they had different default values and so we needed two separate variables.  I'll also reformat this as you suggested!